.github/workflows/update-templates-to-examples.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           script: |
-            github.issues.createComment({
+            github.rest.issues.createComment({
```

Comment:
their API changed?

Reply:
Yeah, since your bump of GH action versions old endpoint was not working anymore